### PR TITLE
Add smoke test for inspection service

### PR DESCRIPTION
**Pull request by vusirikala**
### Description

Add smoke test to check that inspection service is live.

### CODE REVIEW

**Review by JoshLind — COMMENTED** [review]
Nice, thanks @vusirikala 😄  Let's wait for this to land so that you can ping the index page in the test -- it'll be cleaner.

File: testsuite/smoke-test/src/inspection_service.rs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+use crate::smoke_test_environment::new_local_swarm_with_aptos;
+use aptos_forge::Swarm;
+
+#[tokio::test]
+async fn test_inspection_service_connection() {
+    let mut swarm = new_local_swarm_with_aptos(1).await;
+    let info = swarm.aptos_public_info();
+
+    let resp = reqwest::get(info.inspection_service_url().to_owned())
```
Comment (JoshLind) [inline testsuite/smoke-test/src/inspection_service.rs:12]: nit: can we add a comment to this? e.g.,:

```
// Ping the inspection service index page and verify we get a successful response
```

**Review by JoshLind — APPROVED** [review]
Awesome! Thanks @vusirikala 🚀

File: testsuite/smoke-test/src/inspection_service.rs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::smoke_test_environment::new_local_swarm_with_aptos;
+use aptos_forge::Swarm;
+
+#[tokio::test]
+async fn test_inspection_service_connection() {
+    let mut swarm = new_local_swarm_with_aptos(1).await;
+    let info = swarm.aptos_public_info();
+
```
Comment (vusirikala) [inline testsuite/smoke-test/src/inspection_service.rs:11]:
```suggestion
// Ping the inspection service index page and verify we get a successful response
```

File: testsuite/smoke-test/src/inspection_service.rs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::smoke_test_environment::new_local_swarm_with_aptos;
+use aptos_forge::Swarm;
+
+#[tokio::test]
+async fn test_inspection_service_connection() {
+    let mut swarm = new_local_swarm_with_aptos(1).await;
+    let info = swarm.aptos_public_info();
+// Ping the inspection service index page and verify we get a successful response
```
Comment (vusirikala) [inline testsuite/smoke-test/src/inspection_service.rs:11]:
```suggestion
    // Ping the inspection service index page and verify we get a successful response
```

File: testsuite/smoke-test/src/inspection_service.rs (hunk 1 above)
Comment (vusirikala) [reply testsuite/smoke-test/src/inspection_service.rs:12]: Done.

**Comment by igor-aptos** [comment]
any reason to add it to all utilities, instead of just getting it directly in that test? do we expect it to be widely used across tests? 

should it be optional, or is inspection service required to be enabled?

**Review by grao1991 — APPROVED** [review]

**Pull request merged**